pt-BR/mail/chrome/messenger/preferences/receipts.dtd

```diff
@@ -1,5 +1,5 @@
 <!ENTITY dialog.title                  "Confirmações de leitura">
-<!ENTITY receiptSettings.label         "Return Receipts">
+<!ENTITY receiptSettings.label         "Confirmações de leitura">
```

Comment:
I agree, but I checked the English file for line #1, and it was also Return Receipts, so I decided to keep the same for consistency. Do you think we should change both? I'd need Sherman's authorization.